app/main.py

```diff
@@ -1,6 +1,31 @@
+import os
+import requests
+
+API_KEY = os.environ["API_KEY"]
+
+CITY = os.environ.get("CITY", "Paris")
```

Comment:
Code Style Issue: According to the checklist, you should use constants for constant values. Here, 'Paris' is a constant value and should be defined as a constant at the top of your file.